src/GitHub.App/Services/RepositoryForkService.cs

```diff
@@ -101,14 +102,18 @@ public IObservable<object> SwitchRemotes(IRepositoryModel destinationRepository,
 
                         await SwitchRemotes(activeRepo, updateOrigin ? destinationRepository.CloneUrl.ToUri() : null,
                             currentOrigin, trackMasterUpstream);
+                    }
 
-                        if (updateOrigin)
-                        {
-                            vsGitExt.RefreshActiveRepositories();
-                        }
+                    if (updateOrigin)
+                    {
+                        vsGitExt.RefreshActiveRepositories();
 
-                        return new object();
+                        var updatedRepository = vsGitExt.ActiveRepositories.FirstOrDefault();
+                        log.Assert(updatedRepository.CloneUrl == destinationRepository.CloneUrl,
```

Comment:
If something went wrong with `RefreshActiveRepositories`, it's possible that `updatedRepository` could be null. Should change this to `updatedRepository?.CloneUrl`. Sorry! 😊

Reply:
No problem, fixed.